pkg/storage/unified/sql/backend.go

```diff
@@ -76,7 +76,7 @@ func NewBackend(opts BackendOptions) (Backend, error) {
 		isHA:                    opts.IsHA,
 		done:                    ctx.Done(),
 		cancel:                  cancel,
-		log:                     log.New("sql-resource-server"),
+		log:                     logging.FromContext(ctx),
```

Comment:
given that the context is not from a request.... this should likely be 
```
logging.DefaultLogger().With("logger", "sql-resource-server")
```

Reply:
the FromContext approach is useful when the logger is shared across a whole request cycle

Reply:
+1 to what Ryan said